sklearn/utils/estimator_checks.py

```diff
@@ -170,8 +173,10 @@ def _yield_regressor_checks(name, regressor):
     yield partial(check_regressors_train, readonly_memmap=True)
     yield check_regressor_data_not_an_array
     yield check_estimators_partial_fit_n_features
+    if tags["multioutput"]:
+        yield check_regressor_multioutput
     yield check_regressors_no_decision_function
-    if not tags["no_validation"]:
+    if not tags["no_validation"] and name not in ["Lars", "LassoLars"]:
```

Comment:
I don't think this is ideal, we should fix the tag in `Lars` instead.

Also, why is it linked to `tags["no_validation"]` ?

Reply:
After fixing mixins this is no longer needed to pass tests. Removing.